ports/vulkan-hpp/portfile.cmake

```
@@ -3,8 +3,8 @@
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO KhronosGroup/Vulkan-Hpp
-    REF ef609a2f77dd1756e672712f264e76b64acdba61 #v1.3.231
-    SHA512 071cf0d321475a55ecb6eed5874a316daac64a62265f05566c112317ae19cc569e7dfb14ceb9e5c6716a004e2cc621c167d0ffe30faf506096f362c1ed55fafd
+    REF 48b4b6f985c00b9028e6c94088a2c90b1da82c9c #v1.3.259
```

```suggestion
    REF "v${VERSION}"
```
We can use the new function to simplify the steps of updating the port even more. :)

---

Done~